src/main/java/dog/pawbook/ui/MainWindow.java

```diff
@@ -21,7 +21,7 @@
  * The Main Window. Provides the basic application layout containing
  * a menu bar and space where other JavaFX elements can be placed.
  */
-public class MainWindow extends UiPart<Stage> {
+public class MainWindow<FooTabController> extends UiPart<Stage> {
```

Comment:
What does the generic parameter do?

Reply:
Was a work in progress, removed for now.